openapi/openapi.yaml

```diff
@@ -40,17 +46,19 @@ paths:
         - Namespaces
       responses:
         '200':
-          description: The requested Namespace
+          $ref: '#/components/responses/Namespace'
```

Comment:
Description of `#/components/responses/description` is the following:

> Result of creating a Namespace

Whereas it's appropriate for `POST` request, it's not in case of `GET` or `PUT`.
Response descriptions should describe the result of an operation.

Maybe it's better to describe response in place, but refer to a `#/components/schemas/Namespace`?

Reply:
I fixed the description for the Namespace response.